generator/sync_ofsteam.hpp

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include "generator/osm_id.hpp"
+
+#include "std/fstream.hpp"
+#include "std/mutex.hpp"
+#include "std/string.hpp"
+#include "std/vector.hpp"
+
+class SyncOfstream
+{
+  ofstream m_stream;
```

Comment:
Could you please move this to the explicitly-defined private section at the end of the class?